lnst/Recipes/ENRT/GeneveLwtTunnelRecipe.py

```diff
@@ -75,18 +82,22 @@ class GeneveLwtTunnelRecipe(
     # TODO: ping over IPv6 does not work yet
     ip_versions = Param(default=("ipv4",))
     carrier_ipversion = ChoiceParam(type=StrParam, choices=set(["ipv4", "ipv6"]))
+    net_ipv4 = IPv4NetworkParam(default="192.168.101.0/24")
+    net_ipv6 = IPv6NetworkParam(default="fc00::/64")
 
     def configure_underlying_network(self, configuration):
         """
         The underlying network for the tunnel consists of the Ethernet
         devices on the matched hosts.
         """
         host1, host2 = self.matched.host1, self.matched.host2
+        ipv4_addr = interface_addresses(self.params.net_ipv4)
+        ipv6_addr = interface_addresses(self.params.net_ipv6)
         for i, device in enumerate([host1.eth0, host2.eth0]):
```

Comment:
You should remove `i` and `enumerate`  in each recipes, unless they are used by something else.

Reply:
Done.